agenthub/codeact_agent/codeact_agent.py

```diff
@@ -199,11 +205,7 @@ def step(self, state: State) -> Action:
         messages = self._get_messages(state)
         params = {
             'messages': self.llm.format_messages_for_llm(messages),
-            'stop': [
-                '</execute_ipython>',
-                '</execute_bash>',
-                '</execute_browse>',
-            ],
+            'stop': self.stop_sequences,
             'temperature': 0.0,
```

Comment:
Curiously, GitHub diff doesn't show `temperature` line as an added line, but I think we removed it from CodeAct in `main`.

Reply:
Yep it appeared below.